uint32 nModeFlags
string strNewImageFilename
---
int32 nRetCode
